back/engines/commercial/multi_tenancy/lib/tasks/core/hourly_jobs.rake

```diff
@@ -6,7 +6,6 @@ namespace :cl2back do
     now = Time.zone.now
     Tenant.creation_finalized.each do |tenant|
       Apartment::Tenant.switch(tenant.schema_name) do
-        AutomatedTransitionJob.perform_later
```

Comment:
I think we don't have the AutomatedTransitionJob now, as we handle the automated transitions with...
```Ruby
class SideFxReactionService
  include SideFxHelper

  def after_create(reaction, current_user)
    InputStatusService.auto_transition_input!(reaction.reactable.reload) if reaction.reactable_type == 'Idea'
```

But how do we now ensure that the correct automated transitions are run for newly created tenants? (which I guess is what this line ensured)

Reply:
@jinjagit You're right, I made a mistake and removed too much code. I'll recover the automated transitions for the hourly job and tenant creation.